apps/minifront/src/components/shared/non-native-fee-warning.tsx

```diff
@@ -119,19 +32,10 @@ export const NonNativeFeeWarning = ({
    * />
    * ```
    */
+  hasStakingToken: boolean;
   wrap?: (children: ReactNode) => ReactNode;
 }) => {
-  const gasPrices = useGasPrices();
-  const stakingTokenMetadata = useStakingTokenMetadata();
-  const shouldRender =
-    !!amount &&
-    hasTokenBalance({
-      source,
-      balancesResponses,
-      gasPrices,
-      stakingAssetMetadata: stakingTokenMetadata.data,
-    });
-
+  const shouldRender = !!amount && !hasStakingToken;
```

Comment:
This means the notice will show also when using an account with neither UM or an alt gas fee token. That feels like it would add confusion right? 

<img width="1103" alt="Screenshot 2024-10-22 at 12 08 34 PM" src="https://github.com/user-attachments/assets/f06533a4-ab90-430b-988b-7bad9982e85b">

Reply:
I see your point, are you suggesting that we display the privacy warning only for accounts without UM, but with an active alternative gas token balance?